src/neynar-api/neynar-api-client.ts

```diff
@@ -1839,7 +1868,7 @@ export class NeynarAPIClient {
    */
   public async fetchReactionsForCast(
     hash: string,
-    types: ReactionsType,
+    types: ReactionsType[],
```

Comment:
This will be a breaking change for existing users

Reply:
reverted 👍🏻